src/CodeGenerators/SettingsGen.Example/Microsoft.Omex.SettingsGen.Example.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SettingsGen\Microsoft.Omex.CodeGenerators.SettingsGen.csproj"  OutputItemType="Analyzer"  ReferenceOutputAssembly="false"  />
```

Comment:
Could you try setting `ReferenceOutputAssembly` to `true`, and maybe then you don't need other refentece on the next line